documentation/concepts.md

```diff
@@ -202,3 +261,9 @@ allows publishers to *enhance* messages by providing additional metadata and
 serializing their messages into [*envelopes*](./envelope_protocol.md). An
 envelope allows publishers to set things like the `AckInbox`, `Key`, `Headers`,
 and other pieces of metadata.
+
+```
+Do not underestimate the need for meta data on your processes. Proper enrichment of meta data will enable future business models served as new products to new
+consumers, e.g. consumer behaviour etc.
+```
+A final note is to read "Designing Event-Driven Systems" by Ben Stopford that can be found on confluent.io for downloading
```

Comment:
Is there a direct link we can provide?

Reply:
It can be requested here: https://www.confluent.io/designing-event-driven-systems/.

Still, seems a bit fishy to put a link to Confluent here which is the company behind Kafka as a Service? 😅

Reply:
As of now Liftbridge wants to be perceived as a better cloud native substitute for Kafka (in my eyes). Then one "pays tribute" to them. At a pivoting point people realise that Liftbridge is actually much better than Kafka, perhaps v2. Then one stops to compare oneself with Kafka. And that link is a good link. Have you read the book?

Reply:
Not yet but I pushed it to my Kindle. 😅

Reply:
There is a better link. http://www.benstopford.com/2018/04/27/book-designing-event-driven-systems/

Reply:
Great, that's much better!